src/libsyntax/feature_gate.rs

```diff
@@ -378,6 +374,8 @@ declare_features! (
     // Allows `..` in tuple (struct) patterns
     (accepted, dotdot_in_tuple_patterns, "1.14.0", Some(33627)),
     (accepted, item_like_imports, "1.14.0", Some(35120)),
+    // Allows using `Self` and associated types in struct expressions and patterns.
+    (accepted, more_struct_aliases, "1.14.0", Some(37544)),
```

Comment:
I don't know.
For `active` features it's the version in which it was implemented. What does this version mean exactly for `accepted` and `removed` features, something else?
I never cared to update it previously.